assets/css/header.css

```diff
@@ -2,52 +2,90 @@
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 25px 15px;
+    padding: 5% 5%;
 }
 .topbar button{
     background: white;
     border: 1px solid #E5E5E5;
-    border-radius: 20px;
+    border-radius: 30px;
     cursor: pointer;
-    width: 100px;
-    height: 25px;
-    font-size: 10px;
+    color: hsl(192, 100%, 9%);
+    width: calc(60px + 12vw);
+    height: calc(30px + 2vw);
+    font-size: calc(12px + 0.3vw);
     font-family: 'Poppins', sans-serif;
     box-shadow: 0px 2px 3px rgba(0, 0, 0, 0.1);
 }
+header .logo_block {
+    text-align: left;
+}
 .img_logo{
-    width: 100px;
+    width: 50%;
 }
 header{
     background: url(../../images/bg-hero-mobile.svg) no-repeat center center / cover;
+    color: hsl(192, 100%, 9%);
     border-bottom: 1px solid rgba(0, 0, 0, 0.1);
-    height: 700px;
     background-color: hsl(193, 100%, 96%);
 }
 header section {
     display: flex;
+    text-align: center;
+    padding: 0 5% 5% 5%;
+    row-gap: 10px;
     flex-direction: column;
-    justify-content: space-evenly;
     align-items: center;
-    height: 40%;
     font-size: 15px;
     font-family: 'Open Sans', sans-serif;
 }
+header .left {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    row-gap: 20px;
+}
 header section h2{
-    font-size: 25px;
     font-family: 'Poppins', sans-serif;
-    font-weight: 700;
-    width: 80%;
-    margin-bottom: 20px;
+    font-size: calc(25px + 1vw);
+    font-weight: 800;
+    /* width: 80%; */
+    /* margin-bottom: 20px; */
```

Comment:
Please it would be better if you remove commented code. This makes your code look messy😌